### PR TITLE
mcos-xfce-edition: init at  8d90e10

### PR DESCRIPTION
**Pull request by LinArcX**
###### Motivation for this change
https://www.xfce-look.org/p/1210386/

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by LinArcX** [comment]
@dtzWill

**Comment by romildo** [comment]
> https://www.xfce-look.org/p/1279806/

It seems that this link (which is about McOS-MJV-Dark-XFCE-Edition) is not for the theme being packaged.

File: pkgs/data/themes/mcos-xfce-edition/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name = "mcos-xfce-edition";
+
+  src = fetchFromGitHub {
+    owner = "paullinuxthemer";
+    repo = "McOS-XFCE-Edition";
+    rev = "8d90e108377e89359e69d83c83099195b6c98f10";
+    sha256 = "0vvag2gakcsd60ypgkz86s7alhddwk2axgq85v7n4hrda21nhswy";
+  };
+
+#  hash is 0vvag2gakcsd60ypgkz86s7alhddwk2axgq85v7n4hrda21nhswy
```
Comment (romildo) [inline pkgs/data/themes/mcos-xfce-edition/default.nix]: Why to keep this commented lines?

Comment (LinArcX) [reply pkgs/data/themes/mcos-xfce-edition/default.nix]: Which lines? Comme?
I will delete them.

Comment (romildo) [reply pkgs/data/themes/mcos-xfce-edition/default.nix]: Yes.

Comment (LinArcX) [reply pkgs/data/themes/mcos-xfce-edition/default.nix]: Done!

**Pull request closed without merging**